bitriseio/bitrise/certificate.go

```diff
@@ -217,12 +217,10 @@ func (client *Client) RegisterIdentity(certificateSize int64) (RegisterIdentityD
 }
 
 // UploadIdentity ...
-func (client *Client) UploadIdentity(uploadURL string, uploadFileName string, outputDirPath string, exportFileName string) error {
+func (client *Client) UploadIdentity(uploadURL string, uploadFileName string, exportFileName string, content io.Reader) error {
 	log.Printf("Upload %s to Bitrise...", exportFileName)
```

Comment:
`exportFileName` is used only in the log statement, however, the logging could be done on the caller side.

In that case, the parameter is not needed and the function is cleaner in terms of the log output it produces